### DISCUSSION TITLE
Embed a web server inside mobile applications (iOS/Android)

### DISCUSSION THREAD

**Discussion by am-kh** [dotnet/maui#2558] [Ideas]
Will we able to embed a web server inside mobile applications like we can do with WPF?

![image](https://user-images.githubusercontent.com/18553382/133759326-88eec1b0-24ee-4f58-a634-6f166d296db9.png)

**Comment by bitbonk** [comment]
I have exactly the same questions. Our use case is to write an offline documentation browser.
